### PR TITLE
Fix upload of menu logo

### PR DESCRIPTION
**Pull request by petersg83**
### What does it do?

Fix https://github.com/strapi/strapi/issues/13762 (allow users to change the logo in the settings)

### Why is it needed?

To fix the bug

### How to test it?

Try to change the logo, see success

### Related issue(s)/PR(s)

https://github.com/strapi/strapi/issues/13762
https://github.com/strapi/strapi/issues/13906

### CODE REVIEW

**Review by vincentbpro — APPROVED** [review]
Nice catch :wink: LGTM

File: yarn.lock [hunk 1]
```diff
@@ -2640,7 +2640,7 @@
   resolved "https://registry.yarnpkg.com/@hutson/parse-repository-url/-/parse-repository-url-3.0.2.tgz#98c23c950a3d9b6c8f0daed06da6c3af06981340"
   integrity sha512-H9XAx3hc0BQHY6l+IFSWHDySypcXsvsuLhgYLUGywmJ5pswRVQJUHpOsobnLYp2ZUaUlKiKDrgWWhosOwAEM8Q==
 
-"@internationalized/number@^3.1.1":
```
Comment (gu-stav) [inline yarn.lock:2643]: This doesn't look good to me 🤔 . Could you rebase on the latest master and regenerate the lockfile? Looks like there is a version mis-match of the design-system dependency.

**Review by gu-stav — CHANGES_REQUESTED** [review]

**Review by gu-stav — APPROVED** [review]
🌞 🚀

**Comment by petersg83** [comment]
waiting for QA :)

**Review by MarionLemaire — APPROVED** [review]
Tested on Getstarted using Chrome, changing logo in EE 3 times and saving them then going back to initial logo : OK 
image

**Pull request merged**